src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -850,7 +850,7 @@ internal override void Complete()
                                 lineContents.Append(bufferContents[row, column].Character);
                             }
 
-                            bufferText.Append(lineContents.ToString().TrimEnd(Utils.Separators.SpaceOrTab));
+                            bufferText.Append(lineContents.ToString().AsSpan().TrimEnd(" \t"));
```

Comment:
" \t" is way less readable than `SpaceOrTab`.

Reply:
It is single usage the const. It seems it makes no sense to convert to a named constant or add a comment.

Reply:
Then don't call `TrimEnd` here. Instead, we should be able to trim it manually after constructing the `StringBuilder`, with a comment of course.

Reply:
It is not hot path. The code is only for transcribing. No optimizations are needed.

Reply:
If so, then there is no need to make this change. We should not make changes just for the sake of using new span APIs.

Reply:
The PR is for removing static constants which was added only because of old .Net API limitations.